libraries/DataFlash/LogStructure.h

```diff
@@ -920,6 +931,10 @@ Format characters in the format string for binary log messages
       "CURR", CURR_FMT,CURR_LABELS }, \
     { LOG_CURRENT2_MSG, sizeof(log_Current), \
       "CUR2", CURR_FMT,CURR_LABELS }, \
+    { LOG_CURRENT_CELLS_MSG, sizeof(log_Current_Cells), \
+      "CUCE", CURR_CELL_FMT, CURR_CELL_LABELS }, \
```

Comment:
I find CUCx to be a really confusing label... I'm not sure what the best one would be but BATx (or BATV before numbering?) might be a bit more intuitive when reviewing logs. (Arguably thats what the CURR one should be, but we can't change it without confusing everyone).